pylib/gyp/xcodeproj_file.py

```diff
@@ -1508,7 +1512,7 @@ def PathHashables(self):
 
         hashables = []
         xche = self
-        while xche != None and isinstance(xche, XCHierarchicalElement):
+        while xche is not None and isinstance(xche, XCHierarchicalElement):
```

Comment:
if `xche` is None then it can not be an instance of `XCHierarchicalElement`. ;-)